hexa/pipelines/models.py

```diff
@@ -126,6 +126,12 @@ class Meta:
 
     objects = PipelineVersionQuerySet.as_manager()
 
+    @property
+    def is_schedulable(self):
+        return len(self.parameters) == 0 or all(
+            [parameter.get("default") for parameter in self.parameters]
```

Comment:
That's not correct. What if it's a boolean ? False is a valid default

Reply:
This is a good example of something that could be tested @cheikhgwane